content/en/cd-as-a-service/setup/quickstart.md

```diff
@@ -0,0 +1,212 @@
+
+## Install the CD-as-as-Service CLI
+
+{{< include "cdaas/install-cli.md" >}}
+
+### Log in with the CLI
+    
+If you've arrived at this guide without an Armory CD-as-a-Service account, that's OK! You can sign up for a free account when you run `armory login`.
+
+```shell
+armory login
+```
+
+Confirm the device code in your browser when prompted. Then return to this guide.    
+
+## Connect your cluster
+
+CD-as-a-Service uses an agent to execute deployments in your Kubernetes cluster. The installation process uses credentials from your `~/.kube/config` file to install the CD-as-a-Service agent.
+
+Run the following command to install an agent in your Kubernetes cluster:
+
+```shell
+armory agent create
+```
+
+You name your agent during the installation process. This guide references that name as `<my-agent-identifier>`.
+
+## Deploy the example app
+
+Armory's [`potato-facts` example app](https://github.com/armory-io/potato-facts-go) is a simple web app. The UI polls the API backend for facts about potatoes and renders them for users.
+
+### First deployment
+
+Your first deployment deploys the following resources into your Kubernetes cluster:
+
+- Two namespaces: `potato-facts-staging` and `potato-facts-prod`
+- In each namespace: the `potato-facts` app and a Kubernetes `Service`
+
+**Deploy**
+
+Run the following command:
+
+```shell
+armory deploy start -f https://go.armory.io/hello-armory-first-deployment --account <my-agent-identifier>
+```
+
+Congratulations, you've just started your first deployment with CD-as-a-Service! 
+
+You can use the link provided by the CLI to observe your deployment's progression in the [CD-as-a-Service Console](https://console.cloud.armory.io/deployments). CD-as-a-Service deploys your resources to `staging`. Once those resources have deployed successfully, CD-as-a-Service deploys to `prod`.
+
+### Second deployment
+
+CD-as-a-Service is designed to help you build safety into your app deployment process. It does so by giving you declarative levers to control the scope of your deployment. 
+
+CD-as-a-Service has four kinds of constraints that you can use to control your deployment:
+
+- Manual Approvals
+- Timed Pauses
+- [Webhooks]({{< ref "cd-as-a-service/tasks/webhook-approval" >}})
+- [Automated Canary Analysis]({{< ref "cd-as-a-service/setup/canary" >}})
+
+You can use these constraints _between_ environments and _within_ environments:
+
+- During your next deployment, you need to issue a manual approval between `staging` and `prod`. 
+- Within the `prod` deployment, CD-as-a-Service creates a 25/75% traffic split between your app versions. CD-as-a-Service waits for your approval before continuing the deployment.
```

Comment:
its 75/25, I checked the strategy file. 
or we should explicitly say, 75% traffic to old version and 25% traffic to new version